mne/io/base.py

```diff
@@ -2058,3 +2059,190 @@ def _check_update_montage(info, montage):
                        "because they are EOG channels use the eog parameter."
                        % str(missing_positions))
                 raise KeyError(err)
+
+
+def read_raw(input_fname, montage=None, eog=[], misc=[], reference=None,
```

Comment:
and you should put `@verbose` at the top here. I don't *think* you need to pass it to the underlying functions, because they should set `Raw.verbose` to the current verbose level, which should be set appropriately by the `@verbose` tag at the top of this function.